client/app/components/ApplicationArea/ApplicationLayout/DesktopNavbar.jsx

```diff
@@ -165,6 +171,8 @@ export default function DesktopNavbar() {
         <Menu.SubMenu
           key="profile"
           popupClassName="desktop-navbar-submenu"
+          role="menuitem"
```

Comment:
Unless I'm misreading it, `SubMenu` [has this role already](https://github.com/react-component/menu/blob/master/src/SubMenu.tsx#L672).

Reply:
Missed that one